remediation/fence-agents-remediation/how-it-works.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: How It Works
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 1
+---
+
+<!-- markdownlint-disable-next-line MD025 -->
+# How does Fence Agents Remediation work?
```

Comment:
This section seems to repeat the same information of fence-agents-remediation.md

Reply:
It indeed includes similar information as in `fence-agents-remediation.md` (mostly in the first paragraph), but it adds a little bit more information about the concept of fencing and fence agents in the second paragraph.

Reply:
I agree with Carlo, this site doesn't add any value... that very little bit which wasn't mentioned yet can be moved to the parent site.

Reply:
(or the other way around...)